test/integration/framework/common.go

```diff
@@ -192,6 +200,21 @@ func ParseFileAsProviderConfig(filepath string) (*gardencorev1alpha1.ProviderCon
 	return &gardencorev1alpha1.ProviderConfig{RawExtension: apimachineryRuntime.RawExtension{Raw: jsonData}}, nil
 }
 
+// ParseFileAsWorker parses a file as a Worker configuration
+func ParseFileAsWorker(filepath string) ([]gardencorev1alpha1.Worker, error) {
+	data, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return nil, err
+	}
+
+	worker := []gardencorev1alpha1.Worker{}
+	err = yaml.Unmarshal(data, &worker)
+	if err != nil {
```

Comment:
You could also do it a little more compact
`if err = yaml.Unmarshal(data, &worker); err != nil {`